lib/workers/repository/update/branch/auto-replace.ts

```diff
@@ -133,7 +133,8 @@ async function checkExistingBranch(
 export async function doAutoReplace(
   upgrade: BranchUpgradeConfig,
   existingContent: string,
-  reuseExistingBranch: boolean
+  reuseExistingBranch: boolean,
+  firstUpdate = true
```

Comment:
Small optimization to not preemptively check if it's the first autoreplace on the branch